microsoft-365/commerce/close-your-account.md

```diff
@@ -85,10 +85,10 @@ You can also <a href="https://docs.microsoft.com/azure/active-directory/authenti
 
 2. Select **Azure Active Directory**.
 
-3. Switch to the directory you want to delete.
+3. Switch to the organization you want to delete.
 
-4. Select **Delete directory**.
+4. Select **Delete Tenant**.
```

Comment:
```suggestion
4. Select **Delete tenant**.
```

Reply:
Not sure about this capitalization. Please confirm that it is correct.

Reply:
@JohanFreelancer9 I hope the screenshot helps here: <img width="891" alt="Screenshot 2020-10-27 at 08 21 03" src="https://user-images.githubusercontent.com/3469443/97269234-7de9c780-182d-11eb-8e22-8e8e730b7cfb.png">